ee_plugin/ui/forms/add_ee_image.py

```diff
@@ -0,0 +1,104 @@
+import json
+from typing import Callable, Optional
+
+import ee
+from qgis import gui
+from qgis.PyQt import QtWidgets
+from qgis.core import QgsMessageLog, Qgis
+
+from ...Map import addLayer
+from ..widgets import build_form_group_box, build_vbox_dialog
+from ..utils import call_func_with_values
+
+
+# TODO: iface is not used
```

Comment:
In `add_feature_collection.py`, we pass in the `iface` to set the `parent` of the form:

https://github.com/gee-community/qgis-earthengine-plugin/blob/9196a2f2ebbb390d8fb39c03b5c85cb423aff4b7/ee_plugin/ui/forms/add_feature_collection.py#L103

I think I had assumed that this was required. I've removed them in 74c9f96